guides/v2.3/graphql/queries/store-config.md

```diff
@@ -339,3 +225,118 @@ The [ProductInterface]({{ page.baseurl }}/graphql/reference/products.html#Produc
   }
 }
 ```
+
+### Extend configuration data
```

Comment:
Move this section to the end and change the heading to H2

Reply:
done